### DISCUSSION TITLE
Request for Video Tutorial on How to Use Paisa.fyi

### DISCUSSION THREAD

**Discussion by expeditus303** [ananthakumaran/paisa#194] [General]
Hello, Paisa.fyi Community and Developers,

I've recently discovered Paisa.fyi and am intrigued by its features and the privacy-first approach. However, I'm finding the initial setup and navigation a bit challenging due to the unique plain text and ledger-based system.

Could we possibly have a video tutorial series? 

It would be immensely helpful for new users like me to see a step-by-step guide on setting up Paisa, importing transactions, setting budgets, and other common tasks. Visual guides could significantly lower the entry barrier and enhance user experience.

Thank you for considering this request!

**Comment by Piyush-KumarSingh** [comment]
Hello bro i was in a similar situation 1 month before, what i did is joined paisa on matrix.org then asked the dev directly what to do if i am a complete noob in this regard.( and subsequently get help from him from matrix.org itself, he is very helpful)
So he told me to read about plain text accounting from beancount documentation or at least the cheat sheet of beancount(imo not dev's)
Now the next step is to configure the configuration file which you can get from paisa.fyi ( dont remember exactly where it is as someone asked him so he gave them)
You need to input your desired currency, your mutual funds etc (if any) and other things which are asked by config file (already there in something.yaml in document windows but need to copy the content in it )
After that using import you can import 1 month csv file like of hdfc etc. Or do things manually with the help of learnings you got from beancount
I have choosen rather hardest approach using templating(got help from developer as well as chatgpt) , because it will automatically do smthng if certain conditions met without needing you to do things manually but in starting it was too hard for me that i almost gave up as earlier i was using firefly
          Using templating language i can do things like if the csv file has lets say 6 columns where one of which is credit then the amount listed there will go to my bank account you can further categorize it like assets:checkingAccount:Income etc. And many more.
Hope it helps.
